galgebra/mv.py

```diff
@@ -1626,19 +1626,19 @@ def Add(sdop1, sdop2):
                 raise TypeError("Neither argument is a Dop instance")
             return Sdop.Add(sdop1, sdop2)
 
-    def __eq__(self, sdop):
-        if isinstance(sdop, Sdop):
-            if self.Ga != sdop.Ga:
-                return False
-            self = Sdop.consolidate_coefs(self)
-            sdop = Sdop.consolidate_coefs(sdop)
-            if len(self.terms) != len(sdop.terms):
-                return False
-            if set(self.terms) != set(sdop.terms):
-                return False
-            return True
+    def __eq__(self, other):
+        if isinstance(other, Sdop):
+            if self.Ga != other.Ga:
+                return NotImplemented
```

Comment:
Learn Python by reviewing Eric's code series 👍

Reply:
Honestly this line probably could have stayed `False`, left from an intermediate change. Where it does make sense is for the `else` branch below.

Reply:
I read the doc and think it's ok to use `NotImplemented`

Reply:
Oh yeah.